create-toolchain

```diff
@@ -1,13 +1,15 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
 cd "$(dirname "$0")"
 
 DARWIN_TOOLS_VERSION="2.2.0"
 
-[[ "$(uname -s)" = "Darwin" ]] && is_mac=true
+if [[ "$(uname -s)" = "Darwin" ]]; then
+	is_mac=1
+fi
```

Comment:
Is there any reason for this change besides personal stylistic preference? If that's the only reason then consider reverting this change to minimize the diff.

Reply:
Are you referring to the true/false -> 1/0 or the one-liner being written out? I wrote out the one-liners for clarity and believe I changed the true/false for my own sake while testing. I can revert both if you'd like.

Reply:
both, ty

Reply:
Reverted bool and one-liner changes.